app/models/vetting_record.rb

```diff
@@ -1,5 +1,8 @@
 class VettingRecord < ActiveRecord::Base
   belongs_to :user
   validates :skill_proficiency, :experience, :confidence, presence: true
-  scope :find_by_user_id, -> (user_id) { find_by(user_id: user_id) }
+
+  def self.find_by_user_id(user_id)
+    VettingRecord.find_by(user_id: user_id)
+  end
```

Comment:
I refactored while debugging to see if it was the problem, then elected to keep it as is since I'm under the impression that class methods are preferred to scopes. Please let me know if that's not the case.